packages/blade/src/components/Accordion/AccordionButton.native.tsx

```diff
@@ -79,8 +79,10 @@ const _AccordionButton = ({ index, icon: Icon, children }: AccordionButtonProps)
       <Icon size="medium" color={iconColor} marginRight="spacing.3" marginY="spacing.2" />
     );
 
-    if (_index && _icon) {
-      throw new Error(`[Blade: Accordion]: showNumberPrefix and icon shouldn't be used together`);
+    if (__DEV__) {
```

Comment:
so when the mode is prod this is evaluated to false and hence the whole block is removed by the minifier?

Reply:
Yes.

Reply:
| Dev bundle | Prod Bundle |
| ------------|-------------|
| <img width="1031" alt="Screenshot 2023-07-26 at 3 32 15 PM" src="https://github.com/razorpay/blade/assets/46647141/c22bc659-1440-45b3-8ca6-d895312b1fa6"> | <img width="826" alt="Screenshot 2023-07-26 at 3 33 35 PM" src="https://github.com/razorpay/blade/assets/46647141/93ff8c79-238e-4931-a507-5a98dcf9f03c"> |

Reply:
yes i had checked out this branch and checked locally :D